docs/ug/usingReports.md

```diff
@@ -126,13 +126,15 @@ Notes:<br>
 <!-- ==================================================================================================== -->
 
 ## Code panel
-<img src="../images/code-panel.png" alt="code panel" width="468">
+<img src="../images/code-panel-new.png" alt="code panel" width="468">
```

Comment:
Any specific reason for not replacing the original image?

Reply:
I would like to check if you are good with the new image.
I will remove the original image if you agree on the new one.